src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java

```diff
@@ -111,6 +118,39 @@ private void processActions(
     }
   }
 
+  public ActionProcessingService getActionProcessingService(Action action) {
+    ActionCase acase = actionCaseRepo.findById(action.getCaseId());
+    SampleUnitDTO.SampleUnitType caseType =
```

Comment:
What if an ActionCase pre-dates this column being there? I guess it would be null. Should there be default behaviour for those via businessActionProcessingService?